src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/ProgramService.kt

```diff
@@ -9,11 +9,33 @@ import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.repository.ProgramServi
 annotation class ProgramServiceDslMarker
 
 @NomisDataDslMarker
-interface ProgramServiceDsl : CourseActivityDslApi
```

Comment:
Removed all DSL API interfaces and moved the abstract methods inline. This aligns with the DSL pattern used elsewhere.